.github/actions/setup-build/action.yml

```diff
@@ -23,6 +23,9 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Init submodules
+      run: git submodule update --init
```

Comment:
```suggestion
      run: git submodule update --init
      shell: bash
```

Reply:
@rm41339 I took the liberaty to add the missing `shell` key. CI complained about it.